views/userpage/index.tx

```diff
@@ -1,103 +1,125 @@
-                : }
-            </ul>
-            
-            <h2 id="userpage-main-issues-title">Pull Requests</h2>
-
-            <ul id="userpage-main-issues">
-                : if $data.pulls {
-                    : for $data.pulls -> $pull {
-                        <li class="userpage-main-pulls__item">
-                            <div class="left pull_title">
-                                <a href="https://github.com/<:$pull.github_repo.full_name:>/pull/<:$pull.number:>">
-                                    : $pull.title
-                                </a>
-                            </div>
-                            <div class="right pull_ia">
-                                <a href="/ia/view/<:$pull.ia_id:>">
-                                    : $pull.ia_name
-                                </a>
-                            </div>
-                            <div class="clear"></div>
-                        </li>
-                    : }
-                 : }
-            </ul>
-            
-            <h2 id="userpage-main-issues-title">Issues</h2>
-
-            <ul id="userpage-main-issues">
-                : if $data.issues {
-                    : for $data.issues -> $issue {
-                        <li class="userpage-main-issues__item">
-                            <div class="left issue_title">
-                                <a href="https://github.com/<:$issue.github_repo.full_name:>/issues/<:$issue.number:>">
-                                    : $issue.title
-                                </a>
-                            </div>
-                            <div class="right issue_ia">
-                                <a href="/ia/view/<:$issue.ia_id:>">
-                                    : $issue.ia_name
-                                </a>
-                            </div>
-                            <div class="clear"></div>
-                        </li>
-                    : }
-                 : }
-            </ul>
-        </div>
-        <span class="clear"></span>
-    </div>
+<body ng-controller="UserPageController">
+<!-- Header -->
+<header>
+  <div class="container">
+    <div class="stats">
+      <div class="fourth">
+	<span>All Instant Answers</span>
+	<h2>{{count.maintained_ias + count.developed_only_ias}}</h2>
+	</div>
+      <div class="fourth">
+	<span>Pull Requests Reviews</span>
+	<h2>{{count.reviewed_prs}}</h2>
+	</div>
+      <div class="fourth">
+	<span>Open Pull Requests</span>
+	<h2>{{count.developed_prs}}</h2>
+	</div>
+      <div class="fourth">
+	<span>Closed Pull Requests</span>
+	<h2>{{count.closed_prs}}</h2>
+	</div>
+      </div>
+</div></header>
+
+<div class="container">
+
+  <!-- Sidebar -->
+  <aside>
+    <div class="profile card">
+      <img src="{{response.gh_data.avatar_url}}" />
+      <div class="content">
+	<h3>{{user.name}}</h3>
+	<div class="gray">
+	  <a href="http://github.com/{{response.gh_data.login}}">@{{response.gh_data.login}}</a>
```

Comment:
@jagtalon This one too (to `https`). Is there a reason for having it as `http` in the first place?